implementations/rust/ockam/ockam_macros/src/lib.rs

```diff
@@ -168,3 +168,18 @@ fn to_compile_errors(errors: Vec<syn::Error>) -> proc_macro2::TokenStream {
 fn to_compile_error(error: syn::Error) -> TokenStream {
     error.to_compile_error().into()
 }
+
+#[proc_macro]
+pub fn migrate(input: TokenStream) -> TokenStream {
```

Comment:
I don't get why this needs to be a macro

Reply:
Because it should be run during the compilation. Sql scripts don't exist during the runtime

Reply:
Why can't we use [include_str](https://doc.rust-lang.org/std/macro.include_str.html) instead?

Reply:
How would you iterate over the files in the directory with include_str?

Reply:
Also, we actually use sqlx macro under the hood, which you also can't do without a macro

Reply:
got it